torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -1843,8 +1843,9 @@ void InitXlaModuleBindings(py::module m) {
                                   py::call_guard<py::gil_scoped_release>());
   m.def("_xla_get_distributed_runtime_service",
         [](int num_nodes) -> std::unique_ptr<xla::DistributedRuntimeService> {
+          std::string port = runtime::sys_util::GetEnvString("COORDINATOR_PORT", "8547");
```

Comment:
Is `COORDINATOR_PORT` documented anywhere?

Reply:
I think this name needs to be more verbose, unfortunately. There are too many coordinators in our system already: TPU slice discovery, multislice discovery, torch.distributed process groups, and `xla::DistributedRuntimeService`. Maybe `XLA_COORDINATOR_PORT` and `XLA_COORDINATOR_ADDR`?

Reply:
> Is COORDINATOR_PORT documented anywhere?

COORDINATOR_PORT is used to overwrite the default port for pjrt's dist runtime service. I added it in response to another comment. Before it, we initialize the dist runtime service at the port 8547 by default. But when this port is not available (the possibility may be low I assume), users wouldn't be able to start the service and would have to recompile torch_xla. In that case, users can overwrite the default port and unblock themselves.

> Is there ever a case where the torch.distributed master will be different than the distributed runtime master?

So far, torch.distributed master and pjrt's dist runtime master can share the same address and use different ports without any issues. So I'll rename COORDINATOR_PORT to XLA_COORDINATOR_PORT.